RELEASE.md

```diff
@@ -314,32 +315,44 @@ After the release is complete do the following:
   latest bundle manifests.
     - Run `IMAGE_TAG=vx.y.z make bundle` from `bpfman-operator`.
     - Manually update the following tags in
-      bundle/manifests/bpfman-operator.clusterserviceversion.yaml (TODO: automate
+      `bpfman-operator/bundle/manifests/bpfman-operator.clusterserviceversion.yaml` (TODO: automate
       this step).
-        - Change :latest to :vx.y.z on for the example image URLs.
-        - Change "containerImage: quay.io/bpfman/bpfman-operator:latest" to
-          "containerImage: quay.io/bpfman/bpfman-operator:vx.y.z".
+        - Change `:latest` to `:vx.y.z` in all the example image URLs:
+        ```yaml
+        "url": "quay.io/bpfman-bytecode/<example>>:latest"
+        ```
+        to
+        ```yaml
+        "url": "quay.io/bpfman-bytecode/<example>:v0.5.6"
+        ```
+        - Change `:latest` to `:vx.y.z` in the bpfman-operator container image reference:
+        ```yaml
+        containerImage: quay.io/bpfman/bpfman-operator:latest
+        ```
+        to
+        ```yaml
+        containerImage: quay.io/bpfman/bpfman-operator:vx.y.z
+        ```
     - Open a PR in the [community-operator][community-operator] repository with the following:
         - Create a new release directory under `operator/bpfman-operator/` named `x.y.z`:
           ```bash
           cd $SRC_DIR/community-operators/operators/bpfman-operator/
           mkdir x.y.z
           ```
-        - Copy `bpfman-operator/bundle/{manifests, metadata, tests}` to the new release
-          directory and copy`bpfman-operator/Containerfile.bundle.openshift` to the new
-          release directory and rename it `bundle.Dockerfile`.
+        - Copy the directories `bpfman-operator/bundle/{manifests, metadata, tests}` and the file
```

Comment:
Is this the step where you copied the artefacts from your local machine into the GitHub releases "page"? (Via drag+drop?)

Reply:
No, that was when we copied files to the release assets (lines 242 and 307).

Reply:
This: 

> - Run `make build-release-yamls` from the `bpfman-operator/` directory, and then
  add the yaml files generated to the release as assets from the GitHub release
  page.

Should this be "as assets **to** the GitGub release page"?

Reply:
You are saying replace "from" with "to"? Yes. I might rewrite it a little.

Reply:
Yep, much clearer. I also appreciate the nod to "(drag and drop)" for Luddites like me.